### ISSUE TITLE
Fallback table needs a language update to reflect presence of non-required attributes

### ISSUE THREAD

**Issue by melanierichards** [w3c/aria#949]
Some updates in the 1.2 milestone involve moving ARIA properties from "required" to "supported" on various roles. An example is `aria-valuemin`, `aria-valuemax` on the `slider` role (#860). With these changes, the language for the table entitled "Fallback values for missing required attributes" in [Section 9.2](https://www.w3.org/TR/wai-aria-1.2/#document-handling_author-errors_states-properties) no longer agrees with its contents, as there are both required and supported properties in the table.

We should either change the language of Section 9.2, or drop the UA requirements for the supported attributes (IIRC we did not have plans to change UA expectations for these). The table title and references would need updating, as well as the "Required Attribute" column header.

**Comment by melanierichards** [comment]
[Required Properties with Default Values wiki page](https://github.com/w3c/aria/wiki/Required-Properties-with-Default-Values) for a reference of the changes in this project

**Comment by scottaohara** [comment]
I'll take a stab at this.

**Comment by melanierichards** [comment]
On the 2019-05-23 call, the group decided to remove the supported but not required attributes from this table. Some of this cleanup will occur as a consequence of new PRs in this project, like #946, but we should keep this issue open to do a cleanup pass.

**Comment by melanierichards** [comment]
Oh I do want to add that it's a little odd to have to look in multiple places to find the repair techniques for supported attributes. So we'll want to at least add some sentence around 9.2 telling UAs to check the roles for implicit values on supported attrs (MAY statement at most).
